mautic/mautic
# Add id of lead in mautic

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | X
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Required: )
#### Description:
I will add the id field in the mapping of the CRM integration but with my code, it'll make that field available all throughout Mautic. If it's possible to not display it in the other parts of mautic, it would be great.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Open a CRM integration with a mapping
2. See the id lead field don't display in the list of mautic field

#### Steps to test this PR:
1. Apply this PR
2. Open a CRM integration with a mapping
3. See the id lead field display in the list of mautic field
